util/templates/includes/switchsocket.tpl

```diff
@@ -1,5 +1,11 @@
 {{ define "switchsocket" }}
 standbypower: {{ .standbypower }}
-{{ if .integrateddevice }}features: ["integrateddevice"]{{ end }}
+{{ if and .integrateddevice .heating }}
```

Comment:
Das geht einfacher in einer Zeile anstatt aller Kombinationen ;)

Reply:
Mach gerne nen Vorschlag. Alle Einzeiler Varianten die ich gefunden habe waren echt umständlich und effektiv schwerer zu lesen.

Reply:
Liste untereinander ist der Vorschlag

Reply:
Ah. Ich war immer beim Einzeiler mit dem doofen Komma wenn mehr Werte. Wir generieren ja yaml (nicht JSON). Ok danke für den Hinweis.